modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/petra/sql/dsl/expression/FilterPredicateFactoryImpl.java

```diff
@@ -69,6 +68,14 @@ public Predicate create(String filterString, long objectDefinitionId) {
 		return null;
 	}
 
+	@Override
+	public Predicate create(String filterString, long objectDefinitionId) {
```

Comment:
Maybe it would be more clear if we just pass the objectRelatedModelsPredicateProvider and we calculate the key inside:

```
private String _getKey(ObjectRelatedModelsPredicateProvider objectRelatedModelsPredicateProvider) {
	return objectRelatedModelsPredicateProvider.getClassName() + StringPool.POUND + objectRelatedModelsPredicateProvider.getObjectRelationshipType();
}
```